src/screens/Vault/SignerAdvanceSettings.tsx

```diff
@@ -93,9 +94,15 @@ const SignersWithRKSupport = [
   SignerType.TAPSIGNER,
   SignerType.SEEDSIGNER,
   SignerType.SPECTER,
+  SignerType.LEDGER,
+  SignerType.TREZOR,
+  SignerType.BITBOX02,
   SignerType.COLDCARD,
+  SignerType.PASSPORT,
 ];
 
+const SignersReqVault = [SignerType.LEDGER, SignerType.TREZOR, SignerType.BITBOX02];
```

Comment:
I think also Passport no?

Reply:
No, passport is functional without the need for globalXpubs